packages/plugin-link-card/README.md

```
@@ -0,0 +1,131 @@
+# wangEditor 链接卡片 插件
+
+[English Documentation](./README-en.md)
+
+## 介绍
+
+[wangEditor-next](https://cycleccc.github.io/docs/) 链接卡片 插件。
+
+![](./_img/demo.png)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)